src/lib.rs

```diff
@@ -9,18 +9,15 @@
 // except according to those terms.
 
 #![crate_type = "lib"]
-
-#![cfg_attr(not(feature = "norustc"), feature(rustc_private))]
-#![cfg_attr(not(feature = "stable"), feature(test))]
-
 #![deny(unused_imports)]
+#![cfg_attr(feature = "unstable", feature(rustc_private))]
 
-#[cfg(not(feature = "norustc"))]
+#[cfg(feature = "unstable")]
 extern crate rustc;
 
 #[cfg(unix)]
 extern crate libc;
-extern crate libtest as test;
```

Comment:
Why the rename to `libtest`?

Reply:
Don't want it to try and link with the local `test` crate that is unstable and broken now.

Reply:
But an `as` import will never change what gets linked. So why not keep `extern crate libtest as test`?